text/rustconf-steering-committee.md

```diff
@@ -0,0 +1,50 @@
+- Feature Name: rustconf-steering-committee
+- Start Date: 12-27-2023
+- RFC PR: 
+- Rust Issue: N/A
+
+# Summary
+
+This RFC proposes creating a RustConf steering committee to represent the Rust Project’s perspectives and priorities throughout the RustConf 2024 planning process. 
+
+# Proposal
+
+The Rust Foundation staff team would like to propose that a RustConf steering committee composed of ~5 Rust Project members be formed with an agreed-upon process of member appointment and scope established.
+
+A Rust Project steering committee for RustConf could be defined as:
+
+A group of ~5 volunteers from within the Leadership Council and the Project Directors responsible for providing input and direction to the Rust Foundation staff team regarding RustConf 2024. The Rust Foundation will lean on the steering committee to advocate for areas of desired involvement besides the traditional UnConference Day and Program Committee. 
```

Comment:
> from within the Leadership Council and the Project Directors

it's probably useful if those people are involved too, but should the group be limited like this? it might limit people with prior conf organizing experience from participating and probably also increases the work load of the council members and project directors.

Reply:
I agree here. In my opinion, it would make sense to have 1-2 PDs, 1-2 council reps, and 0-2 others.

Reply:
Taking a step back, the most important point is that the program committee reflect a cross cut of the Rust Project and the wider Rust community. We have a mechanism in the Project for reflecting "Project will" that does not require polling all members of the Project: the Council. Unfortunately, we don't have such a mechanism for the wider community, and it seems like it would be an extremely large amount of work in and of itself to attempt to determine such a mechanism. 

With all of that said, I think this RFC should not layout who forms the Steering Committee but rather who _selects_ the Steering Committee membership and what the criteria for that selection are. This allows for flexibility to pick Steering Committee members who fulfill the prescribed criteria without artificially limiting the pool of potential candidates.

## Straw-man Proposal

My straw-man proposal would thus be the following:

### Who selects the steering committee?

The Rust Project Leadership Council is responsible for choosing the steering committee based on the criteria below.

### Criteria for the Committee as a whole
* 6-9 members total
* No more than 2 members may share the same affiliation (using the definition of affiliation used by the Council already)
* A simple majority must be Project members
* TODO: ideally we'd have some criteria for demographic diversity but putting such criteria into strict quantatative terms can be difficult.

### Criteria for individual members
* Must have the time and energy to engage in the process
* Must not be currently under moderation team sanction
* Must have some qualification that speaks towards their knowledge of Rust and the Rust community including but not limited to Project membership, authorship of well known ecosystem crate(s), prominent Rust author/speaker/blogger/etc., organizer of another Rust conference, etc.